bigquery/google/cloud/bigquery/client.py

```diff
@@ -49,7 +49,7 @@
 from google.cloud.bigquery.table import _row_from_mapping
 
 
-_DEFAULT_CHUNKSIZE = 1048576  # 1024 * 1024 B = 1 MB
```

Comment:
Sorry, I don't really understand what's going on in this PR. Marking as needs changes so it doesn't merge.

Reply:
@tswast Sorry, I was just using that as an example to check circleCI issues.  You are right, I forgot to mark it as Do Not Merge.

Reply:
@tswast just ignore this PR.  I'll revert it back to just the `snippets_test` to `snippets` later.